main_ru.tex

```diff
@@ -0,0 +1,24 @@
+% !TeX spellcheck = en_US
+\documentclass[12pt]{article}
+
+% Language setting
+\usepackage[T1]{fontenc}
+\usepackage[russian]{babel}
+
+\def\sections{sections/translated/ru}
+\def\lang_header_adjustment{0px}
+
+\newcommand{\version}{
+  Версия 1.1
+}
+
+\newcommand{\intro}{
+  Эта книга поддерживается сообществом в \href{https://github.com/Heegu-sama/Homm3BG}{репозитории на GitHub}.
+  Вы можете внести свой вклад в разработку, внести изменения или исправить ошибки.
+  Если вы хотите просто оставить отзыв, сделайте это в \href{https://boardgamegeek.com/thread/3235221/rule-book-rewrite-project}{оригинальной ветке BoardGameGeek}.
+}
+\newcommand{\pageshorthand}{стр. }
+
```

Comment:
```suggestion

\newcommand{\addsectionru}[2]{
  \vspace*{-5em}
  \hspace*{-1em}
  \makebox[0pt][l]{
  \raisebox{-\totalheight}[0pt][7pt]{
    \begin{tikzpicture}
      \draw (0, 0) node[inner sep=0] {\includegraphics[width=\linewidth, height=0.2\linewidth]{\layout/section_heading.jpg}};
      \draw (-6.2, 0) node {\includegraphics[width=0.125\textwidth]{#2}};
    \end{tikzpicture}
    }
  }
  \begin{fullwidth}[leftmargin=0.16\textwidth]
    \begin{center}
      \vspace*{\lang_header_adjustment}
      \color{antiquewhite} \section*{\MakeUppercase{\textbf{#1}}}
      \cleardoublepage\phantomsection\addcontentsline{toc}{section}{\protect\numberline{}#1}
    \end{center}
  \end{fullwidth}
  \vspace{1.75em}
}

```

Reply:
Maybe instead of the new command with -ru suffix, make this `\newcommand{\addsection}`, but replace the one in metadata.tex to `\providecommand`?

Reply:
Yes, good idea, didn't know about `providecommand`.